site/src/App/Seo/AppMeta.tsx

```diff
@@ -26,20 +33,21 @@ export function AppMeta() {
           rel="icon"
           type="image/png"
           sizes="16x16"
-          href="https://seek-oss.github.io/braid-design-system/favicon-16x16.png"
+          href={`https://seek-oss.github.io/braid-design-system/${faviconFileName}-16x16.png`}
```

Comment:
Noticed that these werent working on the preview site, because they are pointing to production urls that dont exist yet.

Wondering these have any need to be absolute rather than path relative and can't think of a reason. Making them path relative means they work in dev and preview deployments too.

```suggestion
          href={`/${faviconFileName}-16x16.png`}
```